src/webgl/3d_primitives.js

```diff
@@ -1995,6 +1995,7 @@ p5.RendererGL.prototype.image = function(
 
   this._pInst.noLights();
 
+  noStroke();
```

Comment:
Right now the tests are still failing, I believe it's because we can't call the global `noStroke()` here in case p5 is running in instance mode. I think calling `this._pInst.noStroke()` should fix it though!